src/Components/Register/Register.js

```diff
@@ -21,59 +21,155 @@ const Register = ({ onSubmit, errorLabel }) => {
     password: '',
     repeatPassword: '',
     alterEgo: '',
-    icon: '' });
+    icon: '',
+    isPwdMatched: false
+  });
 
   const handleChange = ({ target }) => {
-    const { name, value } = target;
+
+  const { name, value } = target;
+
+  let checkPwdValidity = {};
```

Comment:
The name checkPwdValidity reffers to an action and this is not an action. Name it pwdMatching instead.

Reply:
Done.